backends/cxxrtl/cxxrtl_backend.cc

```diff
@@ -628,6 +628,22 @@ std::string escape_cxx_string(const std::string &input)
 	return output;
 }
 
+std::string basename(const std::string &filepath)
+{
+#ifndef _WIN32
```

Comment:
Nit: maybe more natural to have the conditional the other way: say what's the separator for `_WIN32`, then handle the rest

Reply:
Updated